WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -836,60 +835,18 @@ - (void)cleanUnusedMediaFileFromTmpDir
     }];
 }
 
-#pragma mark - Networking setup, User agents
```

Comment:
Just following up to my comment on the PR. We should find a way to keep the functionality in `setupUserAgent`, `useDefaultUserAgent` and `useAppUserAgent`.